src/magnetic_spacegroup.c

```diff
@@ -581,8 +582,8 @@ static Symmetry *get_space_group_with_magnetic_symmetry(
 
     /* (a, b, c) = (a_prim, b_prim, c_prim) @ tmat */
     prim_sym = prm_get_primitive_symmetry(tmat, sym, symprec);
-
-    *spacegroup = spa_search_spacegroup_with_symmetry(prim_sym, symprec);
+    *spacegroup =
+        spa_search_spacegroup_with_symmetry(prim_sym, unit_lat, symprec);
```

Comment:
Ping to @lan496, I have modified `spa_search_spacegroup_with_symmetry` to accept basis vectors. Originally, unit basis vectors are set inside `spa_search_spacegroup_with_symmetry`. This change affect to the call here.